Sources/Modules/Myth/TheFirefly/Projectiles/FlowLightMissile.cs

```diff
@@ -29,6 +29,7 @@ public override void SetDefaults()
 	public override void AI()
 	{
 		Player player = Main.player[Projectile.owner];
+		int holdMana = player.statManaMax - 8;
```

Comment:
This variable is unused

Reply:
And there still has so much unused lines in this file. Those lines should be cleaned before this pull request being merged.